partner_priority/data/partner_priority_data.xml

```diff
@@ -0,0 +1,24 @@
+<!-- Copyright 2019 Patrick Wilson <patrickraymondwilson@gmail.com>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo noupdate="1">
+    <record id="res_partner_priority_low" model="partner.priority">
+        <field name="name">Low</field>
+        <field name="description">Low Priority</field>
+        <field name="sequence">1</field>
+    </record>
+    <record id="res_partner_priority_normal" model="partner.priority">
+        <field name="name">Normal</field>
+        <field name="description">Normal Priority</field>
+        <field name="sequence">3</field>
+    </record>
+    <record id="res_partner_priority_elavated" model="partner.priority">
```

Comment:
There is a spelling error in the xml id: elavated instead of elevated.

Reply:
Edited.